src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -86,9 +86,8 @@ public override async ValueTask<FormulaValue> Visit(TableNode node, SymbolContex
                 var arg = await child.Accept(this, context);
                 args[i] = arg;
             }
-
-            // This is always a single column table
-            var tableValue = new InMemoryTableValue(node.IRContext, StandardTableNodeRecords(node.IRContext, args, forceSingleColumn: true));
+            
+            var tableValue = new InMemoryTableValue(node.IRContext, StandardTableNodeRecords(node.IRContext, args, forceSingleColumn: false));
```

Comment:
Is the forceSingleColumn flag needed anywhere else or can it be cleaned up?

Reply:
Used in LibraryMath for Sequence